docs/pages/machine-id/getting-started.mdx

```diff
@@ -0,0 +1,120 @@
+editor                                                            user:list,create,read,update,delete,...
+```
+
+The above output is from a fresh install of Teleport with the default roles, your cluster may have additional roles. In our example, let’s assume you want to give the bot `access` role to allow it to connect to SSH servers within your cluster.
+
+```bash
+$ tctl bots add robot --roles=access
+```
+
+## Step 3/4. Start Machine ID
+
+Now start Machine ID using the `tbot` binary.
+
+```
+$ tbot start \
+   --data-dr=/var/lib/teleport
+   --destination-dir=./machine-id \
+   --token=00000000000000000000000000000000 \
+   --ca-pin=sha256:1111111111111111111111111111111111111111111111111111111111111111 \
+   --auth-server=auth.example.com:3025
+```
+
+Replace the following fields with values from your own cluster.
+
+* `token`: Replace this with the token that was output by the `tctl bots add` command
+* `ca-pin` Replace this with the CA Pin that was output by the `tctl bots add` command
+* `auth-server`: Replace this with the address of your Teleport Auth server.
+<Details scope={["cloud"]} scopeOnly={true}>
+* `auth-server`: Replace this with the address of your Teleport auth server server.
+</Details>
+<Details scope={["oss","enterprise"]} scopeOnly={true}>
+* `auth-server`: Replace this with the address of your Teleport Cloud proxy server.
+</Details>
+
+Once Machine ID is successfully started, let’s investigate the `machine-id` directory to see what was written to disk.
+
+```
+$ tree ./machine-id
+machine-id
+├── key
+├── key.pub
+├── known_hosts
+├── ssh_config
+├── sshcacerts
+├── sshcert
+├── tlscacerts
+└── tlscert
+
+0 directories, 8 files
+```
+
+This directory contains private key material, SSH certificates, x509 certificates, and OpenSSH configuration to make it easy to integrate Machine ID with external tooling.
+
+## Step 4/4. Use Machine ID
+
+To use Machine ID, find a host that is registered within your cluster to connect to using `tsh ls`. You might see output like the following on your own system.
+
+
+```bash
+$ tsh ls
+Node Name Address        Labels                        
+--------- -------------- ----------------------------- 
+node-name  127.0.0.1:3022 arch=x86_64,group=api-servers 
+```
+
+To use Machine ID with OpenSSH integration, simply run the following command to connect to `node-name`.
+
+```
+ssh -F ~/machine-id/ssh_config root@node-name
+```
+
+Now you can replace any invocations of `ssh` with the above command for automation to use Machine ID.
```

Comment:
I think it's worth mentioning that `tbot` is still running in the background and periodically renewing these certificates.